docs/continuing_runs.md

```diff
@@ -0,0 +1,204 @@
+matches with the newly created folder. Then, proceed by editing the
+configuration file as discussed in the previous examples. Finally,
+
+```
+haddock3 new-config-file-run2.cfg --restart N
+```
+
+where `N` is the step you wish to start from, as explained previously. Notably,
+you do not need to copy the whole folder when copying the original run. Instead,
+you can copy only the modules folders you wish to maintain plus the `data`
+folder.
+
+# Extend an unknown run
+
+HADDOCK3 allows you to extend a run without previous knowledge of the run
+configuration. For example, you have several runs for which you **now** want to
+perform a CAPRI evaluation at the last step. How to do this?
+
+Create a new configuration file containing just the `caprieval` module, or
+whatever and many modules you desire. For example:
+
+```toml
+[caprieval]
+reference_fname = "reference-structure.pdb"
+```
+
+Now, run the configuration file by giving the `--restart` flag the path of the
+previous run directory:
+
+```
+haddock3 caprieval.cfg --restart <previous-run-directory>
+```
+
+In this case, the configuration file does not need to contain any information
+about the molecules used as input or the previous modules. HADDOCK3 will just
+execute the new modules on top of the previous run.
+
+# Use a module as a seed for a new run
+
+Imagine you want to take a specific module of a successful run as the starting
+point of a new run. Copy the `data`, the `topoaa`, and the desired module to a
+new folder. You can also use the `topoaa` as the initial module. At this point,
+you can start the new run using both strategies we have detailed so far:
+
+1) edit the original configuration file keeping only the information for the
+modules you used as seed and editing the `run_dir` parameter to the new folder,
+and execute:
+
+```
+haddock3 new-config.cfg --restart 2
+```
+
+If your seed module is directly the `topoaa` module, use `--restart 1`.
+
+2) Create a new configuration file containing only the parameters for the
+modules you want to run and execute:
+
+```
+haddock3 new-config.cfg --restart <run_dir>
+```
+
+where the `run_dir` is the new directory containing the seeding modules.
+
+# Final considerations
+
+When extending or continuing a run from a new configuration file, all the
+modules' folders and their content will be updated to the corresponding position
+and leading zeros. For example, the new takes steps 0 and 4 from a run with more
+than 10 modules (note the leading double digit numbers):
+
+```
+00_topoaa/
+04_flexref/
+data/
+```
+
+If you extend this run with a series of `caprieval`, `emref`, `caprieval`, and
+`clustfcc`, the final output will be:
+
+```
+0_topoaa/
+1_flexref/
+2_caprieval/
+3_emref/
+4_caprieval/
+5_clustfcc/
+```
+
+Where `0_topoaa` and `1_flexref` are the original folders renamed. The files
+within these folders were also edited to ensure paths compatibility and the
+consistency of paths and the run as a whole.
```

Comment:
Additional consideration:

Provide an option to copy specific modules from an existing run to a new run. E.g.

I want to take over modules 0 1 4 and 8. Copy those and renumber then sequentially (probably needed)

Reply:
Okay. That's a good functionality. I will add it.